Add tasksupport capability for filter operation

Mashashi opened the pull request.

Commit:
Add tasksupport capability for filter operation

flicken commented:
Hi Mashashi , 

Thanks for the pull request!  I've added TaskSupport support for all operations in e48c43416c99745ee838ae0ecfc2c3ee089b4da6. 

FYI: I've changed the order of arguments and added a default value to allow for backwards compatibility.

Cheers,
  Brian

Mashashi commented:
I am the one who thanks for your good work! This stuff should be shipped
with scala! Best regards.

Hi Mashashi ,

Thanks for the pull request! I've added TaskSupport support for all
operations in e48c434
https://github.com/tim-group/iterata/commit/e48c43416c99745ee838ae0ecfc2c3ee089b4da6.

FYI: I've changed the order of arguments and added a default value to allow
for backwards compatibility.

Cheers,
Brian

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
https://github.com/tim-group/iterata/pull/1#issuecomment-225457218, or mute
the thread
https://github.com/notifications/unsubscribe/ADr22CV9SSdjbQ6KDeJZsJomJsnhbLB4ks5qLGX5gaJpZM4IzzOc
.
